### DISCUSSION TITLE
Can SimpleIdServer support Device Flow?

### DISCUSSION THREAD

**Discussion by steve2bi** [simpleidserver/SimpleIdServer#504] [Ideas]
Can SimpleIdServer support Device Flow? or it is in the plan?

**Comment by simpleidserver** [comment]
Hello,

It is not yet planned, but we can include this feature in the next release ( 3.0.4)

Kr,

Sid

**Comment by simpleidserver** [comment]
The device flow is supported by SimpleIdServer.
The feature will be available in the next release `4.0.0` : Ticket #505 

KR,

SID
